rust-runtime/smithy-eventstream/fuzz/fuzz_targets/mutated_read.rs

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#![no_main]
+
+use bytes::{Buf, BufMut};
+use crc32fast::Hasher as Crc;
+use libfuzzer_sys::{fuzz_mutator, fuzz_target};
+use smithy_eventstream::frame::{Header, HeaderValue, Message};
+use smithy_types::Instant;
+
+fn mutate(data: &mut [u8], size: usize, max_size: usize) -> usize {
```

Comment:
a comment about this fuzz mutation strategy would be good

Reply:
Added a comment.